docs/CONTRIBUTING_ZH.md

```diff
@@ -0,0 +1,96 @@
+# 如何参与 native pulsar go client 的开发
```

Comment:
probably better add the file to `docs/zh/CONTRIBUTING.md`? so we can have 1:1 mapping between english docs and chinese docs.

Reply:
good idea!